nautobot/dcim/api/serializers.py

```diff
@@ -199,8 +199,7 @@ class ConnectedEndpointSerializer(PathEndpointModelSerializerMixin):
 
 class RegionSerializer(NautobotModelSerializer, TreeModelSerializerMixin):
     url = serializers.HyperlinkedIdentityField(view_name="dcim-api:region-detail")
-    # TODO: why was this disabled? What is needed before we can re-enable it?
```

Comment:
Oh this was done by me for the generic create view when I was working on disabling the nested serialization so that the JSON schema generation worked correctly using PKs and not expecting an entire new object to be created in place in the create form.